### ISSUE TITLE
Arrows

### ISSUE THREAD

**Issue by govorukhincom** [kosbarts/Commissioner#49]
Hey! Firstly, thanks for a splendid font. Curious if you were planning to add arrow glyphs (← ↑ → ↓)? Working on an interface with quite some arrows, and fallback arrows feel slightly off.

If there's a way I can contribute them to the font — I'd love to.

**Comment by kosbarts** [comment]
Hi @govorukhincom . Thank you very much for your kind words. 

Do you only need the four arrows? I could add just these if that's only it. 

But the SIL Open License allows you to fork it and make changes as you like provided you credit the designer and keep the same license. Check it out here: https://github.com/kosbarts/Commissioner/blob/master/OFL.txt

**Comment by govorukhincom** [comment]
Yeah, just four arrows would be great! Thanks in advance. 💖 Didn't know the font is SIL Open License'd — thanks for a suggestion.

**Comment by kosbarts** [comment]
I have added arrows in version 1.011 that I just pushed.
